source/Halibut/ServiceModel/RequestCancellationTokens.cs

```diff
@@ -11,28 +11,37 @@ public RequestCancellationTokens(CancellationToken connectingCancellationToken,
         {
             ConnectingCancellationToken = connectingCancellationToken;
             InProgressRequestCancellationToken = inProgressRequestCancellationToken;
-        }
-
-        public CancellationToken ConnectingCancellationToken { get; set; }
-        public CancellationToken InProgressRequestCancellationToken { get; set; }
 
-        public CancellationToken LinkedCancellationToken => LazyLinkedCancellationToken.Value;
-
-        Lazy<CancellationToken> LazyLinkedCancellationToken => new (() =>
```

Comment:
Removed the Lazy as the LinkedCancellationToken is always referenced so the Lazy is always created.

Reply:
Reason: Simplification to remove any possibility of retention due to the referenced objects of the lazy